app/tests/fixtures/lti_tool_registration_fixture.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Note, no tests in here yet

Reply:
The fixtures are for `Lti13TestCase::test_findIssuers()`.

Reply:
my bad (didn't realize it was a fixture at the time!)